test/python/test_transport.py

```diff
@@ -146,7 +146,20 @@ def test_add_species_multi(self):
         self.assertNear(gas1.thermal_conductivity, gas2.thermal_conductivity)
         self.assertArrayNear(gas1.multi_diff_coeffs, gas2.multi_diff_coeffs)
 
-    def test_species_visosities(self):
+    def test_high_pressure_chung_viscosity(self):
+        state = 520, 6e7, 'NH3:1.0'
+
+        gas1 = ct.Solution('gri30.yaml')
+        gas1.transport_model = 'high-pressure-chung'
+        gas1.TPX = state
+
+        # Values from Poling et al. (2001), example 9-12 for ammonia
+        experimental_viscosity = 466e-7 # 466 micropoise = 466e-7 Pa s
+
+        error = abs(gas1.viscosity - experimental_viscosity) / experimental_viscosity
+        self.assertTrue(error <= 0.05) # Error should be less than 5%
```

Comment:
Prefer the `pytest` style asserts:

```suggestion
        # Error should be less than 5%
        assert gas1.viscosity == pytest.approx(experimental_viscosity, 0.05)
```